aws/data_source_aws_db_subnet_group.go

```diff
@@ -0,0 +1,93 @@
+package aws
+
+import (
+	"fmt"
+	"log"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/rds"
+	"github.com/hashicorp/terraform/helper/schema"
```

Comment:
```suggestion
	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
```
here we should reference the schema framework as part of terraform-plugin-sdk instead